.github/workflows/security.yml

```diff
@@ -110,8 +123,21 @@ jobs:
       - name: Flag snap scanning alerts
         run: |
           cd /home/runner
-          jq '.runs[].tool.driver.rules[] |= (.shortDescription.text |= "Snap scan - " + .)' ${{ matrix.version }}-stable.sarif > tmp.json
-          mv tmp.json ${{ matrix.version }}-stable.sarif
+          # Modify the SARIF file in-place by prefixing each rule's shortDescription.text with "Snap scan - "
+          cat <<< $(jq '.runs[].tool.driver.rules[] |= (.shortDescription.text |= "Snap scan - " + .)' ${{ matrix.version }}-stable.sarif) > ${{ matrix.version }}-stable.sarif
+
+      - name: Tag KEV alerts
```

Comment:
I didn't quite find where this step was altering the `.sarif` file that is fed back into GH.